File: minElementFromArray.java
class minElementFromArray {
        public int FindMin(int a[]) {
                int min = a[0];
                for (int i = 0; i < a.length; i++) {

                        if (min > a[i]) {
                                min = a[i];
                                // return min;
                        }
                }
                return min;
        }


public static void main(String[] args) {
        int a[]={22,10,5,16,77,89};
        System.out.println(new minElementFromArray(). FindMin(a));
}
}

File: JavaScript/MinElementFromArray.js
class minFromArray{
        findMin(a){
                let min =a[0];
                for(let i=0 ;i<a.length;i++){
                        if(min>a[i]){
                                min = a[i];
                        }
                }
        return min;
        }
}

const a=[22,17,5,465,77];
console.log(new minFromArray().findMin(a));